src/language-js/types/estree.d.ts

```diff
@@ -44,12 +45,12 @@ type PrettierEsComment = Comment & {
   leading?: boolean;
 };
 
-export interface PrettierEsNode {
+export type PrettierEsNode = Node & {
```

Comment:
I did try this, but failed on my machine.

Reply:
This is good!

Reply:
Can we export this as `Node`?

Reply:
Yes I have done that now, but it probably needs to be fixed better.